janus.ggo

```diff
@@ -37,3 +37,4 @@ option "token-auth" A "Enable token-based authentication for all requests" flag
 option "token-auth-secret" - "Secret to verify HMAC-signed tokens with, to be used with -A" string typestr="randomstring" optional
 option "event-handlers" e "Enable event handlers" flag off
 option "no-webrtc-encryption" w "Disable WebRTC encryption, so no DTLS or SRTP (only for debugging!)" flag off
+option "cwd-path" H "Working directory for Janus daemon process (default=/)" string typestr="path" optional
```

Comment:
Please put this where the other daemon-related property is. The file is also missing an endline, apparently, whichshould be fixed. Besides, any new property added here must be added to the README (output of `janus -h`) and the `janus.1` file (man page) as well.

Reply:
Missing does added here https://github.com/meetecho/janus-gateway/pull/1804/commits/14d0818ac47bdcf5bca7d866eeb5fd0fd5b361e2